src/nupic/regions/KNNClassifierRegion.py

```diff
@@ -84,7 +84,17 @@ def getSpec(cls):
             required=False,
             regionLevel=True,
             isDefaultInput=False,
-            requireSplitterMap=False)
+            requireSplitterMap=False),
+
+          sequenceIdIn=dict(
```

Comment:
As discussed, please remove sequenceIdIn from KNNClassifierRegion and then I can merge. Thanks!